.github/workflows/vulnerability-scans.yml

```diff
@@ -1,6 +1,8 @@
 # GitHub Actions CI workflow that runs vulnerability scans on the application's Docker image
 # to ensure images built are secure before they are deployed.
 
+# RUN A TEST
+
```

Comment:
Change looks good - just need to revert this

```suggestion
```

Reply:
oh thanks